test/links.c

```diff
@@ -13716,6 +13716,107 @@ external_link_with_committed_datatype(hid_t fapl, hbool_t new_format)
     return FAIL;
 } /* end external_link_with_committed_datatype() */
 
+/*-------------------------------------------------------------------------
+ * Function:    external_link_public_macros
+ *
+ * Purpose:     Test public macros for external links
+ *
+ * Return:      Success:        0
+ *              Failure:        -1
+ *-------------------------------------------------------------------------
+ */
+static int
+external_link_public_macros(hid_t fapl, hbool_t new_format)
+{
+    hid_t       fid = -1;               /* File ID */
+    H5L_info2_t linfo;                  /* Link information */
+    char        objname[NAME_BUF_SIZE]; /* Object name */
+    char        filename1[NAME_BUF_SIZE];
+    char        filename2[NAME_BUF_SIZE];
+    unsigned   *flags = NULL; /* External link flags, packed as a bitmap */
```

Comment:
This needs to be a plain `unsigned flags = NULL;` so that it can be passed to H5Lunpack_elink_val as `&flags`.

Reply:
Fixed.